internal/collector/pgadmin.go

```diff
@@ -10,120 +10,120 @@ import (
+						// https://opentelemetry.io/docs/specs/otel/logs/data-model-appendix/#appendix-b-severitynumber-example-mappings
+						// https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/-/pkg/ottl/contexts/ottllog#enums
+						`set(severity_number, SEVERITY_NUMBER_DEBUG)  where severity_text == "DEBUG"`,
+						`set(severity_number, SEVERITY_NUMBER_INFO)   where severity_text == "INFO"`,
+						`set(severity_number, SEVERITY_NUMBER_WARN)   where severity_text == "WARNING"`,
+						`set(severity_number, SEVERITY_NUMBER_ERROR)  where severity_text == "ERROR"`,
+						`set(severity_number, SEVERITY_NUMBER_FATAL)  where severity_text == "CRITICAL"`,
+					},
 				},
 			},
-		},
-	}
-
-	// If there are exporters to be added to the logs pipelines defined in
-	// the spec, add them to the pipeline. Otherwise, add the DebugExporter.
-	exporters := []ComponentID{DebugExporter}
-	if spec != nil && spec.Logs != nil && spec.Logs.Exporters != nil {
-		exporters = slices.Clone(spec.Logs.Exporters)
-	}
-
-	otelConfig.Pipelines["logs/pgadmin"] = Pipeline{
-		Extensions: []ComponentID{"file_storage/pgadmin_data_logs"},
-		Receivers:  []ComponentID{"filelog/pgadmin"},
-		Processors: []ComponentID{
-			"resource/pgadmin",
-			"transform/pgadmin_log",
-			ResourceDetectionProcessor,
-			LogsBatchProcessor,
-			CompactingProcessor,
-		},
-		Exporters: exporters,
-	}
-
-	otelConfig.Pipelines["logs/gunicorn"] = Pipeline{
-		Extensions: []ComponentID{"file_storage/pgadmin_data_logs"},
-		Receivers:  []ComponentID{"filelog/gunicorn"},
-		Processors: []ComponentID{
-			"resource/pgadmin",
-			"transform/pgadmin_log",
-			ResourceDetectionProcessor,
-			LogsBatchProcessor,
-			CompactingProcessor,
-		},
-		Exporters: exporters,
-	}
+		}
+
+		// If there are exporters to be added to the logs pipelines defined in
+		// the spec, add them to the pipeline. Otherwise, add the DebugExporter.
+		exporters := []ComponentID{DebugExporter}
+		if spec != nil && spec.Logs != nil && spec.Logs.Exporters != nil {
+			exporters = slices.Clone(spec.Logs.Exporters)
+		}
+
+		otelConfig.Pipelines["logs/pgadmin"] = Pipeline{
+			Extensions: []ComponentID{"file_storage/pgadmin_data_logs"},
+			Receivers:  []ComponentID{"filelog/pgadmin"},
+			Processors: []ComponentID{
+				"resource/pgadmin",
+				"transform/pgadmin_log",
+				ResourceDetectionProcessor,
+				LogsBatchProcessor,
+				CompactingProcessor,
+			},
+			Exporters: exporters,
+		}
+
+		otelConfig.Pipelines["logs/gunicorn"] = Pipeline{
+			Extensions: []ComponentID{"file_storage/pgadmin_data_logs"},
+			Receivers:  []ComponentID{"filelog/gunicorn"},
+			Processors: []ComponentID{
+				"resource/pgadmin",
+				"transform/pgadmin_log",
+				ResourceDetectionProcessor,
+				LogsBatchProcessor,
+				CompactingProcessor,
+			},
+			Exporters: exporters,
+		}
 
-	otelYAML, err := otelConfig.ToYAML()
-	if err == nil {
-		configmap.Data["collector.yaml"] = otelYAML
+		otelYAML, err := otelConfig.ToYAML()
```

Comment:
🤔 Does this shadow `err`?

❓ Is this big hunk just changing indentation? If so, the early return might be preferrable.